cmake/unifex_flags.cmake

```diff
@@ -27,3 +27,15 @@ if(CXX_MEMORY_RESOURCE_HAVE_PMR)
 else()
   set(UNIFEX_NO_MEMORY_RESOURCE TRUE)
 endif()
+
+# Probe for libUring support
+find_package(LibUring COMPONENTS Experimental Final)
```

Comment:
Do we need the Experimental/Final parameters here?

These are added to the find_package() calls above to indicate that they should search for both experimental/final versions of the standard-library features. I don't think that really applies to liburing.